static/usage/select/basic/multiple-selection/angular.md

```diff
@@ -0,0 +1,7 @@
+```html
+<ion-select placeholder="Select all fruits that apply" multiple>
```

Comment:
```suggestion
<ion-select placeholder="Select all fruits that apply" [multiple]="true">
```
Multiple is a property not an attribute so we need to always provided "true" (even though this will technically work. Angular apps with strict mode enabled will complain about this)

Reply:
This is also true for all of the other examples. The compiler may not complain, but it's still not correct usage.